sway/commands/move.c

```diff
@@ -354,6 +354,7 @@ static bool container_move_in_direction(struct sway_container *container,
 				// Container will swap with or descend into its neighbor
 				container_move_to_container_from_direction(container,
 						target, move_dir);
+				container_update_representation(container);
```

Comment:
Would we run into issues if `container_move_to_container_from_direction` called `container_update_representation` in the `list_swap` case? There's precedent, e.g. `container_move_to_workspace` calls `container_update_representation` internally when necessary, so it'd be nice to have the entire class of functions behave the same.

Reply:
Makes sense and should work.